### ISSUE TITLE
Adding padding to container div moves the checkbox off

### ISSUE THREAD

**Issue by yongelee** [lokesh-coder/pretty-checkbox#31]
https://imgur.com/a/pdp1g

One of these checkboxes has added padding but the checkbox does not align properly. How do I make the area bigger without ruining it?

**Comment by lokesh-coder** [comment]
Hi @yongelee ,

Thanks for attaching a screenshot. But, I am bit unclear about the area you mentioned. You want the green box to be smaller and have more space around it?

Check this please, 
https://jsbin.com/rewugaz/1/edit?html,css,output

**Comment by yongelee** [comment]
No...the whitespace inside the border of the div should be fatter with padding. But when padding is added, the position of the checkbox moves like the picture.
